devel/simdjson/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+PortGroup        github 1.0
+PortGroup        cmake 1.1
+
+github.setup     simdjson simdjson 3.1.7 v
+revision         0
```

Comment:
pleas add "github.tarball_from ???" And set it toe either "releases" (preferred if upstream generates those), otherwise to "archive". This will very likely require you to update the "checksums" as well.